examples/DotNetCore/ChatApp/ChatApp/CompletionConfiguration.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace ChatApp
 {
-    internal class ModelConfiguration
+    internal class CompletionConfiguration

```

Comment:
I'm thinking if we want to go all the way with `ChatCompletionConfiguration`

Reply:
Sure, why not :)